Sources/VLCSettingsTableHeaderView.swift

```diff
@@ -0,0 +1,70 @@
+/*****************************************************************************
+ * VLCSettingsTableHeaderView.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import UIKit
+
+class VLCSettingsTableHeaderView: UITableViewHeaderFooterView {
+    let headerHeight: CGFloat = 1.0
```

Comment:
Do we need to expose this outside of this class? If not we should consider "privatising" it 😄